activerecord/lib/active_record/secure_token.rb

```diff
@@ -14,13 +14,13 @@ module ClassMethods
       #   # Schema: User(token:string, auth_token:string)
       #   class User < ActiveRecord::Base
       #     has_secure_token
-      #     has_secure_token :auth_token, length: 36
+      #     has_secure_token :auth_token, length: 36, prefix: "auth_"
```

Comment:
Should `length` take the prefix into consideration? If not (i believe is the case), should we clarify that in the docs?